content/docs/user-guide/basic-concepts/dvc-cache.md

```diff
@@ -3,7 +3,7 @@ name: 'DVC Cache'
 match: ['DVC cache', cache, caches, cached, 'cache directory', caching]
 tooltip: >-
   The DVC cache is a hidden storage (by default in `.dvc/cache`) for files and
-  directories tracked by DVC, and their different versions. Learn more about its
-  structure
-  [here](/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory).
+  directories tracked by DVC, and their different versions. For efficiency, it
```

Comment:
@jorgeorpinel can we please avoid non substantial fixes please? (do it in a separate PR?)

Reply:
> Besides the index file, what the other changes are about (too many files touched

Not sure how this got up to 10 files heh. I think it started with links and then make some fixes around that (like quotes -> admons, typos, etc.)

I'll separate them, but we can focus on reviewing .../data-management/index.md here.

Reply:
and diagrams

Reply:
> Not sure how this got up to 10 files heh 

Actually, 4 of them are images. Only 2 files had mostly-unrelated changes, but OK: I extracted those to https://github.com/iterative/dvc.org/pull/4060. Pls approve/merge if possible.